openassessment/xblock/static/sass/oa/utilities/_shame.scss

```diff
@@ -2,6 +2,10 @@
 // ====================
 
 // NOTES: use for any bad-form/orphaned scss that knowingly violate edX FED architecture/standards (see - http://csswizardry.com/2013/04/shame-css)
+#seq_content .wrapper--xblock .openassessment h3.staff__grade__title {
```

Comment:
My aim was to sync up heading styles across LMS and CMS for the main content area after removing H1s and H2s as options from the visual editor. Due to the overall messiness of the platform styles, I had to either use an !important to ensure things were consistent, or be more specific in targeting.

We should discuss this, perhaps with Christine. Without my specificity, styles aren't really cohesive. With the specificity then Eric must be even more specific. Not a win-win in either scenario.